tests/ExposeAllAttributes.py

```diff
@@ -32,14 +32,14 @@ class ExposeAllAttributes(Framework.TestCase):
     def testAllClasses(self):
         authenticatedUser = self.g.get_user()
         namedUser = self.g.get_user("nvie")
-        repository = authenticatedUser.get_repo("PyGithub")
+        repository = self.g.get_repo("wrecker/PyGithub")
```

Comment:
Why are you changing this test too? I can see you're struggling with some test issues, but this isn't the way.

Reply:
The change to last_modified for ContentFile has introduced an additional API request. This breaks the recorded requests in the replay data for this test. The only way to fix this is to re-build the testcase and its replaydata. And I cannot recreate the replaydata with the same repo since I don't have access to the PyGithub repo.